components/tabs/src/tabs.udl

```diff
@@ -19,7 +19,7 @@ dictionary RemoteTab {
     string title;
     sequence<string> url_history;
     string? icon;
-    u64 last_used;
```

Comment:
Technically a breaking API change, so please also add a note to the changelog.

Reply:
Will do! I do wonder if this will mark a major upgrade when cutting a build given that this is more of a patch since v81 turned the API to be u64 instead of i64 being passed over the wire.

Reply:
I think it was i64 before this uniffi work, right?